### PR TITLE
Convert to class and style

### PR DESCRIPTION
**Pull request by qwibbler**
Add calculator UI as per requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Additional description about the project and its features.
 - Node.js
 - React
 
-## Live Demo
+<!-- ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://livedemo.com) -->
```
Comment (Ghiftee) [inline README.md]: - [x] Kindly remove commented code. If you do not have the live link now, you can add it at another time.

File: README.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ To get a local copy up and running follow these simple example steps.
 - Open `index.html` in the `build` directory.
 - Enjoy!
 
-### Usage
+<!-- ### Usage
 
 ### Run tests
 
-### Deployment
+### Deployment -->
```
Comment (Ghiftee) [inline README.md]: - [x] Kindly remove commented code. If you do not have the necessary information now, you can add it at another time.

File: README.md [hunk 3]
```diff
@@ -61,11 +61,11 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<!-- ## Acknowledgments
 
 - Hat tip to anyone whose code was used
 - Inspiration
-- etc
+- etc -->
```
Comment (Ghiftee) [inline README.md]: - [x] Kindly remove commented code. If you do not have the necessary information now, you can add it at another time.

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (Ghiftee) [inline src/App.js]: - [x] Please make your App.js component class-based.

File: src/components/Calculator.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Button from './Button';
+// import Row from './Row';
```
Comment (Ghiftee) [inline src/components/Calculator.js]: - [x] Kindly remove commented code. It makes your code cleaner.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @qwibbler 

Good job working on the project so far! 
✔️  Your app matches the design given
✔️  There are no linter errors

However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by omar-labana — APPROVED** [review]
Hi @qwibbler, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**